Lambdas/GHHooks/EventHandler/Handlers/DocsIssuer.swift

```diff
@@ -64,32 +65,38 @@ struct DocsIssuer {
 
     func getPRsRelatedToCommit() async throws -> [SimplePullRequest] {
         let response = try await context.githubClient.repos_list_pull_requests_associated_with_commit(
-            .init(path: .init(
-                owner: repo.owner.login,
-                repo: repo.name,
-                commit_sha: commitSHA
-            ))
+            .init(
+                path: .init(
+                    owner: repo.owner.login,
+                    repo: repo.name,
+                    commit_sha: commitSHA
+                )
+            )
         )
 
         guard case let .ok(ok) = response,
-              case let .json(json) = ok.body else {
+            case let .json(json) = ok.body
```

Comment:
While I approve of it moving the `else` to its own line, I actively dislike it actively un-lining-up the individual conditions.